2020/2020-11-19-the-crown.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: post
+title: The Crown
```

Comment:
What should be the title @alkidel? What season was this?

Reply:
The Crown, season 4: politics, lust, and loathing